### ISSUE TITLE
FileInstall extracting empty files

### ISSUE THREAD

**Issue by SaifAqqad** [AutoHotkey/Ahk2Exe#83]
For some reason, after the latest version of Ahk2Exe (v1.1.33.10), `FileInstall` is now extracting empty files.

As a workaround, I'm now using `AddResource` and these two functions to extract the resource:

```ahk

getResourcePtr(resource){
    if hMod := DllCall("GetModuleHandle", "UInt", 0, "PTR")
        if hRes := DllCall("FindResource", "UInt", hMod, "Str", resource, "UInt", 10, "PTR")
            if hData := DllCall("LoadResource", "UInt", hMod, "UInt", hRes, "PTR")
                if pData := DllCall("LockResource", "UInt", hData, "PTR")
                    return { ptr: pData
                            , size: DllCall( "SizeofResource", "UInt", hMod, "UInt", hRes, "PTR")}
}

extractResources(resource){
    if(FileExist(resource) || !(res:= getResourcePtr(resource)))
        return
    file:= FileOpen(resource, "w")
    file.RawWrite(res.ptr, res.size)
    file.Close()
}

```

**Comment by TAC109** [comment]
Could you give me an exact copy of the failing command, please. Also whether the FileInstall'ed file actually was inserted into the compiled program? (Any other details you can supply could be helpful as well.)

Thanks

**Comment by SaifAqqad** [comment]
Here's a video showing the `FileInstall` and compiling it with 1.1.33.10

https://user-images.githubusercontent.com/47293197/132801173-fc78c154-af23-43f1-86fe-1a1ec396968d.mp4

[Link to the compiled file](https://cdn.discordapp.com/attachments/781788491225759744/885747653722386492/MicMute.exe)

**Comment by SaifAqqad** [comment]
Here's a video showing the same thing but with 1.1.33.09

https://user-images.githubusercontent.com/47293197/132801739-c1a69b61-71db-4e6d-bb6a-380f33e675de.mp4

[Link to the compiled file](https://cdn.discordapp.com/attachments/781788491225759744/885750250881548298/MicMute.exe)

**Comment by SaifAqqad** [comment]
Btw I only replaced the Ahk2Exe.exe file, so they're both using the same .bin

**Comment by TAC109** [comment]
This is fixed in v1.1.34.00 Beta 3. 

Cheers

**Comment by TAC109** [comment]
Oops, I introduced another bug with Beta 3.
Now fixed properly in 1.1.34.00 Beta 4.

All the best
